pages/configuration/advanced.md

```diff
@@ -23,7 +23,7 @@ The `config.txt` is located in the root of the `resin-boot` partition, and you c
 
 #### Modifying `config.txt` **remotely**
 
-If your device has an Agent (Supervisor) version above 1.0.0 (which you can check on the "Agent version" entry in the device dashboard), it has support for modifying the values in `config.txt` remotely using config variables which behave similarly to [environment variables](/management/env-vars) in that you can set application-wide values (for all devices in an application) or device-specific ones.
+If your device has an Agent (Supervisor) version above 1.0.0 (which you can check on the "Agent version" entry in the [device dashboard](https://dashboard.resin.io)), it has support for modifying the values in `config.txt` remotely using config variables which behave similarly to [environment variables](/management/env-vars) in that you can set application-wide values (for all devices in an application) or device-specific ones.
```

Comment:
You should be able to use `{{ dashboardUrl }}` instead of repeating the actual URL (but please verify that it works)